framework/auth/core.py

```diff
@@ -56,7 +56,6 @@ def string_required(value):
 
 
 def validate_history_item(item):
-    string_required(item.get('institution'))
```

Comment:
Does it have to be this way? By the time the data reach the server, the data should be valid, correct? In other words, `institution` should always be present in the payload sent to the server.